src/parsers/manifest/dash/parse_mpd.ts

```diff
@@ -320,11 +320,18 @@ function parseCompleteIntermediateRepresentation(
     }
   }
 
+  const isLastPeriodKnown =
+    !isDynamic ||
+    (mpdIR.attributes.minimumUpdatePeriod === undefined &&
```

Comment:
The fact that we consider the last Period unknown by checking the `minimumUpdatePeriod` attributes of the manifest is a little bit mystic for me. I guess it's written in the spec, but it could be worth mentioning it as a comment here no?

Reply:
You're right.

Yes it's basically a DASH-IF thing

Reply:
Commented